src/logic/contracts/safeContracts.js

```diff
@@ -10,19 +10,46 @@ import { calculateGasOf, calculateGasPrice } from '~/logic/wallets/ethTransactio
 import { ZERO_ADDRESS } from '~/logic/wallets/ethAddresses'
 
 export const SENTINEL_ADDRESS = '0x0000000000000000000000000000000000000001'
+export const multiSendAddress = '0xB522a9f781924eD250A11C54105E51840B138AdD'
```

Comment:
I know this is replicated from what Nico did on another branch. But can we keep the approach of using all caps snake cased constants?

```javascript
export const MULTI_SEND_ADDRESS = '0xB522a9f781924eD250A11C54105E51840B138AdD'
```

Reply:
done